<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow{
	color:WHITE;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
	background-color: rgb(53, 53, 53);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="Output_Area">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>281</width>
      <height>651</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTextEdit{
    border: 4px solid #555;
    border-radius: 20px;
	color:WHITE;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
	background-color: rgb(53, 53, 53);
    padding: 5px;
}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Code_Area">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>961</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTextEdit{
    border: 4px solid #555;
    border-radius: 20px;
	color:WHITE;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
	background-color: rgb(53, 53, 53);
    padding: 5px;
}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="console">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>506</y>
      <width>961</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTextEdit{
    border: 4px solid #555;
    border-radius: 20px;
	color:WHITE;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
	background-color: rgb(53, 53, 53);
    padding: 5px;
}
</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="Open_File"/>
    <addaction name="Save_File"/>
    <addaction name="actionClose_File"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="ScannerTab"/>
    <addaction name="ParserRun"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Open_File">
   <property name="text">
    <string>Open File      Ctrl+O</string>
   </property>
  </action>
  <action name="Save_File">
   <property name="text">
    <string>Save File        Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose_File">
   <property name="text">
    <string>Close File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="ScannerTab">
   <property name="text">
    <string>Scanner      Ctrl+R</string>
   </property>
  </action>
  <action name="ParserRun">
   <property name="text">
    <string>Parser</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
